model/resource/container.yaml

```
@@ -44,8 +47,19 @@ groups:
           K8s defines a link to the container registry repository with digest `"imageID": "registry.azurecr.io
           /namespace/service/dockerfile@sha256:bdeabd40c3a8a492eaf9e8e44d0ebbb84bac7ee25ac0cf8a7159d25f62555625"`.
 
-          OCI defines a digest of manifest.
+          The ID is assinged by the container runtime and can vary in different environments.
+          Consider using `oci.manifest.digest` if it is important to identify the same
+          image in different environments/runtimes.
         examples: ['sha256:19c92d0a00d1b66d897bceaa7319bee0dd38a10a851c60bcec9474aa3f01e50f']
+      - id: image.repo_digests
+        type: string[]
+        brief: >
+         Repo digests of the container image as provided by the container runtime.
+        note: >
+          [Docker](https://docs.docker.com/engine/api/v1.43/#tag/Image/operation/ImageInspect) and
+          [CRI](https://github.com/kubernetes/cri-api/blob/c75ef5b473bbe2d0a4fc92f82235efd665ea8e9f/pkg/apis/runtime/v1/api.proto#L1237-L1238)
+          report those under the `RepoDigests` field.
+        examples: [ 'sha256:e4ca62c0d62f3e886e684806dfe9d4e0cda60d54986898173c1083856cfda0f4', 'sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b' ]
```

nit: I think it should be full ones that include repo URL
```suggestion
        examples: [ 'example@sha256:e4ca62c0d62f3e886e684806dfe9d4e0cda60d54986898173c1083856cfda0f4', 'internal.registry.example.com:5000/example@sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b' ]
```

---

Good catch! Fixed at e5a3293, thank you!